input/ig.xml

```diff
@@ -2122,5 +2138,9 @@
 			<code value="special-url"/>
 			<value value="http://terminology.hl7.org.au/ValueSet/coverage-type-extended"/>
 		</parameter>
+		<parameter>
```

Comment:
Addition of a duplicate code fragment - needs to be removed.

Reply:
Removed duplication.